doc/support.rst

```diff
@@ -90,13 +90,9 @@ client works fine: http://webchat.freenode.net
 Documentation resources
 =======================
 
-This documentation is relative to |release|. Documentation for other
-versions can be found here:
+This documentation is relative to |release|. Documentation for
+other versions can be found `here
+<http://scikit-learn.org/dev/documentation.html>`_.
```

Comment:
I think `http://scikit-learn.org/dev/versions.html`  would be more direct.

Reply:
Yes, thanks!